src/dtclient/connection_info.go

```diff
@@ -6,6 +6,13 @@ import (
 	"github.com/pkg/errors"
 )
 
+// CommunicationHost => struct of connection endpoint
+type CommunicationHost struct {
+	Protocol string // nolint:unused
+	Host     string // nolint:unused
+	Port     uint32 // nolint:unused
+}
```

Comment:
shouldn't this be in the `oneagent_connection_info.go` as its only relevant for the oneagent connection info

Reply:
true